protocols/luci-proto-modemmanager/htdocs/luci-static/resources/protocol/modemmanager.js

```diff
@@ -117,11 +117,22 @@ return network.registerProtocol('modemmanager', {
 		o.value('ipv6', _('IPv6 only'));
 		o.default = 'ipv4v6';
 
+		s.taboption('general', form.Value, 'signalrate', _('Signal refresh rate'));
```

Comment:
Hey @valexi7 , can you change it to something like this?  You should validate the input where possible.

```
o = s.taboption('general', form.Value, 'signalrate', _('Signal refresh rate'), _('How often to get the signal strength metrics from the modem, in seconds.'));
o.datatype = "uinteger";
```